lib/Controller/RoomController.php

```diff
@@ -253,13 +253,23 @@ protected function formatRoom(Room $room, Participant $participant = null): arra
 		foreach ($participants['users'] as $userId => $data) {
 			$user = $this->userManager->get((string) $userId);
 			if ($user instanceof IUser) {
-				$participantList[(string) $user->getUID()] = [
+				$participantList[(string)$user->getUID()] = [
 					'name' => $user->getDisplayName(),
 					'type' => $data['participantType'],
 					'call' => $data['inCall'],
+					'sessionId' => $user->getSessionId(),
 				];
+
+				try {
+					$participant = $room->getParticipant((string)$user->getUID());
```

Comment:
this does one query per user...
I will try to fix this and put it in the first response